File: Submissions/#0580_div2/A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  vector<int> num(1001);

  int n;
  cin >> n;

  vector<int> a(n);
  for (int i = 0; i < n; i++) {
    cin >> a[i];
    num[a[i]]++;
  }

  int m;
  cin >> m;

  vector<int> b(m);
  for (int i = 0; i < m; i++) {
    cin >> b[i];
    num[b[i]]++;
  }

  bool flg = false;

  for (int i = 0; i < n; i++) {
    for (int j = 0; j < m; j++) {
      int c = a[i] + b[j];
      if (num[c] == 0) {
        cout << a[i] << " " << b[j] << endl;
        flg = true;
        break;
      }
    }
    if (flg) {
      break;
    }
  }

}

File: Submissions/#0580_div2/A/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {
    
    Scanner sc = new Scanner(System.in);
    
    int[] num = new int[1001];

    int n = sc.nextInt();
    int[] a = new int[n];
    for (int i = 0; i < n; i++) {
      a[i] = sc.nextInt();
      num[a[i]]++;
    }

    int m = sc.nextInt();
    int[] b = new int[m];
    for (int i = 0; i < m; i++) {
      b[i] = sc.nextInt();
      num[b[i]]++;
    }

    sc.close();

    boolean flg = false;

    for (int i = 0; i < n; i++) {
      for (int j = 0; j < m; j++) {
        int c = a[i] + b[j];
        if (num[c] == 0) {
          System.out.println(a[i] + " " + b[j]);
          flg = true;
          break;
        }
      }
      if (flg) {
        break;
      }
    }
    
  }

}